ql/src/java/org/apache/hadoop/hive/ql/stats/BasicStatsTask.java

```diff
@@ -258,7 +268,7 @@ private int aggregateStats(Hive db) {
         Partish p;
         partishes.add(p = new Partish.PTable(table));
 
-        BasicStatsProcessor basicStatsProcessor = new BasicStatsProcessor(p, work, conf, followedColStats);
+        BasicStatsProcessor basicStatsProcessor = new BasicStatsProcessor(table, p, work, followedColStats);
```

Comment:
I don't think we need these table callse - you may simply use `Partish#getTable` to get access to the table object later